mne/epochs.py

```diff
@@ -536,13 +536,13 @@ def _detrend_offset_decim(self, epoch, verbose=None):
         # Detrend
         if self.detrend is not None:
             picks = pick_types(self.info, meg=True, eeg=True, stim=False,
-                               ref_meg=False, eog=False, ecg=False,
+                               ref_meg=False, eog=False, ecg=False, seeg=True,
                                emg=False, exclude=[])
             epoch[picks] = detrend(epoch[picks], self.detrend, axis=1)
 
         # Baseline correct
         picks = pick_types(self.info, meg=True, eeg=True, stim=False,
-                           ref_meg=True, eog=True, ecg=True,
+                           ref_meg=True, eog=True, ecg=True, seeg=True,
                            emg=True, exclude=[])
```

Comment:
BTW. What are these reference MEG channels used for? Here they are not detrended, but baselining is applied.